corehq/apps/fixtures/download.py

```diff
@@ -32,6 +32,7 @@ def prepare_fixture_download(table_ids, domain, task, download_id):
     return expose_cached_download(
         file.getvalue(),
         60 * 60 * 2,
+        "xlsx",
```

Comment:
I'm pretty sure that `expose_cached_download`'s third positional arg is mimetype—and anyway it would probably be clearer if you wrote `suffix='.xlsx'`. (Also I think it's `'.xlsx'` with a period.)